integration/client/helpers_darwin_test.go

```diff
@@ -0,0 +1,58 @@
+// +build darwin
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package containerd
+
+import (
+	"context"
+
+	"github.com/containerd/containerd/containers"
+	"github.com/containerd/containerd/oci"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+const newLine = "\n"
+
+func withExitStatus(es int) oci.SpecOpts {
+	return func(_ context.Context, _ oci.Client, _ *containers.Container, s *specs.Spec) error {
+		// TODO: use appropriate command instead of dummy (hello) command
+		s.Process.Args = []string{"hello"}
+		return nil
+	}
+}
```

Comment:
Why do we have this "hello" while some others tests are skipped?

Reply:
because currently available runtime (i.e., runu) only supports a single process, there are several limitations on various tests.

If a test only involves a single process (doesn't use fork syscall), then this hello program is used.